app/views/shops/new.html.erb

```diff
@@ -0,0 +1,37 @@
+<h1>Input information for a new shop</h1>
```

Comment:
The name should reflect what the page does. Shop creation - Input information for a new shop
is more suitable